Jenkinsfile

```diff
@@ -198,7 +198,8 @@ pipeline {
 
 def buildImages(){
   log(level: 'INFO', text: "buildImages with ${MAKEFILE} for ${PLATFORM}")
-  withGoEnv(){
+  // this version is used only for preparing the templating.
+  withGoEnv(version: '1.20.7'){
```

Comment:
The important golang versions are defined in the dockerfiles. 

This version is only required for running the go template.go in the host.

`env.GO_VERSION` is used for knowing when a new release needs to happen